snapcraft/internal/build_providers/_base_provider.py

```diff
@@ -53,6 +47,9 @@ def __init__(self, *, project, echoer) -> None:
             self.snap_filename = "{}_{}.snap".format(
                 project.info.name, project.deb_arch
             )
+        self.provider_project_dir = os.path.join(
+            BaseDirectory.xdg_data_home, "snapcraft", "projects", project.info.name
```

Comment:
Perhaps `os.path.join(xdg.BaseDirectory.save_data_path("snapcraft"), "projects", project.info.name)` ?

Reply:
Sure, why not? Is there a difference?